entrypoint.sh

```diff
@@ -4,29 +4,33 @@ export RELATIVE=
 export ANDROID=
 export BASELINE=
 
-if [ "$INPUT_KTLINT_VERSION" = "latest" ] ; then
-  curl -sSL https://api.github.com/repos/pinterest/ktlint/releases/latest \
-    | grep "browser_download_url.*ktlint\"" \
-    | cut -d : -f 2,3 \
-    | tr -d \" \
-    | wget -qi -\
-    && chmod a+x ktlint \
-    && mv ktlint /usr/local/bin/
+if [ "$INPUT_KTLINT_VERSION" = "latest" ]; then
+  echo "Downloading latest..."
+  content=$(curl -L https://api.github.com/repos/pinterest/ktlint/releases/latest)
+  echo "$content"
```

Comment:
@renatoabreu11 finally got the actual error:
https://github.com/ScaCap/action-ktlint/actions/runs/3773090106/jobs/6414367317
```
{"message":"API rate limit exceeded for 172.177.36.112. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}
```